MAUI/ComboBox/UI-Customization.md

```diff
@@ -173,4 +173,528 @@ comboBox.MaxDropDownHeight = 150;
+        ColumnDefinition colDef1 = new ColumnDefinition() { Width = 48 };
+        ColumnDefinition colDef2 = new ColumnDefinition() { Width = 220 };
+        RowDefinition rowDef = new RowDefinition() { Height = 50 };
+        grid.ColumnDefinitions.Add(colDef1);
+        grid.ColumnDefinitions.Add(colDef2);
+        grid.RowDefinitions.Add(rowDef);
+
+        Image image = new();
+        image.HorizontalOptions = LayoutOptions.Center;
+        image.VerticalOptions = LayoutOptions.Center;
+        image.Aspect = Aspect.AspectFit;
+        image.SetBinding(Image.SourceProperty, ("ProfilePicture"));
+        Grid.SetColumn(image, 0);
+
+        StackLayout stack = new();
+        stack.Orientation = StackOrientation.Vertical;
+        stack.Margin = new Thickness(15, 0, 0, 0);
+        stack.HorizontalOptions = LayoutOptions.Start;
+        stack.VerticalOptions = LayoutOptions.Center;
+        Grid.SetColumn(stack, 1);
+
+        Label label = new();
+        label.SetBinding(Label.TextProperty, "Name");
+        label.FontSize = 14;
+        label.TextColor = Colors.Red;
+        label.VerticalOptions = LayoutOptions.Center;
+        label.HorizontalTextAlignment = TextAlignment.Start;
+        label.Opacity = .87;
+
+        Label label1 = new();
+        label1.SetBinding(Label.TextProperty, "Designation");
+        label1.FontSize = 12;
+        label1.TextColor = Colors.Green;
+        label1.VerticalOptions = LayoutOptions.Center;
+        label1.HorizontalTextAlignment = TextAlignment.Start;
+        label1.Opacity = .54;
+
+        stack.Children.Add(label);
+        stack.Children.Add(label1);
+
+        grid.Children.Add(image);
+        grid.Children.Add(stack);
+
+        return new ViewCell { View = grid };
+    });
+
+    EmployeeTemplateSelector employeeTemplateSelector = new EmployeeTemplateSelector();
+    employeeTemplateSelector.EmployeeTemplate1 = employeeTemplate1;
+    employeeTemplateSelector.EmployeeTemplate2 = employeeTemplate2;
+
+    SfComboBox comboBox = new SfComboBox()
+    {
+        HeightRequest = 34,
+        WidthRequest = 280,
+        BindingContext = employee,
+        ItemsSource = employee.Employees,
+        DisplayMemberPath = "Name",
+        Placeholder = "Enter an employee",
+        TextMemberPath = "Name",
+        ItemTemplate = employeeTemplateSelector,
+    };
+
+    this.Content = comboBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox ItemTemplateSelector](Images/UICustomization/TemplateSelector.png)
+
+## Completed Event
+
+The [Completed]() event is raised when the user finalizes the text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode by pressing return key on the keyboard.The handler for the event is a generic event handler, taking the `sender` and `EventArgs`(the `EventArgs` value is `string.Empty`):
```

Comment:
The [Completed]() event is raised when the user finalizes the text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode by pressing return key on the keyboard. The handler for the event is a generic event handler, taking the `sender` and `EventArgs`(the `EventArgs` value is `string.Empty`):